akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -425,6 +425,7 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `text/css`                  = txt("css", "css")
   val `text/csv`                  = txt("csv", "csv")
   val `text/html`                 = txt("html", "htm", "html", "htmls", "htx")
+  val `text/markdown`             = txt("markdown", "md")
```

Comment:
cool, it needs to be replicated to `akka.http.javadsl.model.MediaTypes` too though